contrib/kind.sh

```diff
@@ -1336,6 +1336,29 @@ function install_kubevirt() {
             kubectl logs --all-containers=true -n kubevirt $p || true
         done
     fi
+    
+    if [ ! -d "./bin" ]
+    then
+        mkdir -p ./bin
+        if_error_exit "Failed to create bin dir!"
+    fi
+
+    if [[ "$OSTYPE" == "linux-gnu" ]]; then
```

Comment:
I guess these two are the only types we support right? and so its not really a new dependency and I think we call `install_kubevirt` only if the kubevirt flag is set so we are good on the default cases...

Reply:
The code is an "adaptation" of https://github.com/ovn-org/ovn-kubernetes/blob/master/contrib/kind.sh#L29, at the end is the same thing different binary.